e2e/case/particleRenderer-emissive.ts

```
@@ -0,0 +1,122 @@
+      },
+    ])
+    .then((resources) => {
+      const fireEntity = createDebrisParticle(engine, <Texture2D>resources[0], <GLTFResource>resources[1],<Texture2D>resources[2]);
+      scene.addRootEntity(fireEntity);
+
+      updateForE2E(engine, 500);
+      initScreenshot(engine, camera);
+    });
+});
+
+function createDebrisParticle(engine: Engine, texture: Texture2D, glTFResource: GLTFResource, emissionTexture: Texture2D): Entity {
```

ESLint

[error] Replace `engine:·Engine,·texture:·Texture2D,·glTFResource:·GLTFResource,·emissionTexture:·Texture2D` with `⏎··engine:·Engine,⏎··texture:·Texture2D,⏎··glTFResource:·GLTFResource,⏎··emissionTexture:·Texture2D⏎`

(prettier/prettier)